demo/kubernetes/rook-storageclass.yaml

```diff
@@ -1,3 +1,18 @@
+apiVersion: rook.io/v1alpha1
+kind: Rookpool
+metadata:
+  name: rook-replicapool
+spec:
+  name: replicapool
```

Comment:
based on this news, i would also rename the spec `namespace` to `clusterNamespace` or `rookClusterNamespace` to illustrate that this arg is to define the namespace that the rook-operator is in and not necessary the namespace where the rookpool resource will live